editor/gui/scene_tree_editor.cpp

```diff
@@ -508,6 +486,36 @@ void SceneTreeEditor::_add_nodes(Node *p_node, TreeItem *p_parent) {
 	}
 }
 
+void SceneTreeEditor::_update_node_tooltip(Node *p_node, TreeItem *p_item) {
+	// Display the node name in all tooltips so that long node names can be previewed
+	// without having to rename them.
+	String tooltip = String(p_node->get_name());
```

Comment:
```suggestion
	String tooltip = p_node->get_name();
```
Unnecessary explicit cast.
(I know it was like that before, but the code can be tweaked when it's moved)